modules/mono/editor/GodotTools/GodotTools.IdeMessaging/ClientHandshake.cs

```diff
@@ -9,7 +9,7 @@ public class ClientHandshake : IHandshake
 
         public string GetHandshakeLine(string identity) => $"{ClientHandshakeBase},{identity}";
 
-        public bool IsValidPeerHandshake(string handshake, out string identity, ILogger logger)
+        public bool IsValidPeerHandshake(string handshake, out string? identity, ILogger logger)
```

Comment:
Seems like this parameter is missing the nullability attribute.

```suggestion
        public bool IsValidPeerHandshake(string handshake, [NotNullWhen(true)] out string? identity, ILogger logger)
```